### ISSUE TITLE
Generic URL ––> archive.org persistent ID/URL ––> Manubot

### ISSUE THREAD

**Issue by RaoOfPhysics** [greenelab/manubot-rootstock#85]
When you cite a news or blog URL, you might want to reference the archive.org snapshop of the URL.

Can the `@url:` identifier send a request to archive.org and get _that_ URL to cite in Manubot?

See blog post: https://medium.com/@RaoOfPhysics/89bd3f2ce0fd

**Comment by agitter** [comment]
Interesting idea @RaoOfPhysics.  I've done this manually for citations but had not considered it within Manubot.  Some sites (like [GitHub](http://web.archive.org/web/20171113122312/https://github.com/greenelab/manubot-rootstock/issues/85)) don't render well in the snapshot, but the content is the most important part.

We would have to think about access dates.  I don't think we would want to update the snapshot every time Manubot builds the manuscript.  I don't recall how archive.org decides whether to make new snapshots if an older one exists.  Will `https://web.archive.org/save/` create a new one every time it is called?

**Comment by RaoOfPhysics** [comment]
Thanks for responding, @agitter. I had the same thought about multiple snapshots, and I'm afraid I don't have an answer!

**Comment by agitter** [comment]
As a test, I was able to create a second archive snapshot of this page: https://web.archive.org/web/*/https://github.com/greenelab/manubot-rootstock/issues/85

Each snapshot has a `YYYYMMDDhhmmss` timestamp so we appear to have the flexibility to create multiple snapshots and cite a specific one.  I don't think we would want to create a new snapshot every time the manuscript builds though.

**Comment by vsmalladi** [comment]
@agitter I agree we would only want to have a snapshot on a tagged version of a manuscript.

**Comment by agitter** [comment]
I noticed in deep review that greycite can identify an archived version of a website in some cases.  Here's one example: https://github.com/greenelab/deep-review/commit/d96eee7056bdc76292e0ed8ac761b79be5064930#diff-a20cf4739729d1d4ca84b2b0ff66d0ceR58830

**Comment by agitter** [comment]
I noticed the [`waybackpy` package](https://pypi.org/project/waybackpy/) that could simplify archiving URL references.
